.jshintrc

```diff
@@ -1,84 +1,16 @@
-    "eqnull"        : false,    // true: Tolerate use of `== null`
-    "es5"           : false,    // true: Allow ES5 syntax (ex: getters and setters)
-    "esnext"        : false,    // true: Allow ES.next (ES6) syntax (ex: `const`)
-    "evil"          : false,    // true: Tolerate use of `eval` and `new Function()`
-    "expr"          : false,    // true: Tolerate `ExpressionStatement` as Programs
-    "funcscope"     : false,    // true: Tolerate defining variables inside control statements"
-    "globalstrict"  : false,    // true: Allow global "use strict" (also enables 'strict')
-    "iterator"      : false,    // true: Tolerate using the `__iterator__` property
-    "lastsemic"     : false,    // true: Tolerate omitting a semicolon for the last statement of a 1-line block
-    "laxbreak"      : false,    // true: Tolerate possibly unsafe line breakings
-    "laxcomma"      : false,    // true: Tolerate comma-first style coding
-    "loopfunc"      : false,    // true: Tolerate functions being defined in loops
-    "multistr"      : false,    // true: Tolerate multi-line strings
-    "proto"         : false,    // true: Tolerate using the `__proto__` property
-    "scripturl"     : false,    // true: Tolerate script-targeted URLs
-    "smarttabs"     : false,    // true: Tolerate mixed tabs/spaces when used for alignment
-    "shadow"        : false,    // true: Allows re-define variables later in code e.g. `var x=1; x=2;`
-    "sub"           : false,    // true: Tolerate using `[]` notation when it can still be expressed in dot notation
-    "supernew"      : false,    // true: Tolerate `new function () { ... };` and `new Object;`
-    "validthis"     : false,    // true: Tolerate using this in a non-constructor function
-
-    // Environments
-    "browser"       : true,     // Web Browser (window, document, etc)
-    "couch"         : false,    // CouchDB
-    "devel"         : true,     // Development/debugging (alert, confirm, etc)
-    "dojo"          : false,    // Dojo Toolkit
-    "jquery"        : false,    // jQuery
-    "mootools"      : false,    // MooTools
-    "node"          : false,    // Node.js
-    "nonstandard"   : false,    // Widely adopted globals (escape, unescape, etc)
-    "prototypejs"   : false,    // Prototype and Scriptaculous
-    "rhino"         : false,    // Rhino
-    "worker"        : false,    // Web Workers
-    "wsh"           : false,    // Windows Scripting Host
-    "yui"           : false,    // Yahoo User Interface
-
-    // Legacy
-    "nomen"         : false,    // true: Prohibit dangling `_` in variables
-    "onevar"        : false,    // true: Allow only one `var` statement per function
-    "passfail"      : false,    // true: Stop on first error
-    "white"         : true,     // true: Check against strict whitespace and indentation rules
-
-    // Custom Globals
-    "predef"        : [ ]       // additional predefined global variables
+    "browser": true,
```

Comment:
If we're using eslint, do we need jshint as well?

Reply:
I first thought the .jshintrc was needed for IDE purposes but there is also eslint support for at least PHPStorm.
So yes, we can drop it.